mmdet3d/models/segmentors/base.py

```diff
@@ -0,0 +1,109 @@
+import mmcv
+import numpy as np
+import torch
+from mmcv.parallel import DataContainer as DC
+from mmcv.runner import auto_fp16
+from os import path as osp
+
+from mmdet3d.core import show_seg_result
+from mmseg.models.segmentors import BaseSegmentor
+
+
+class Base3DSegmentor(BaseSegmentor):
+    """Base class for 3D segmentors.
+
+    The main difference with `BaseSegmentor` is that we modify the keys in
+    data_dict and use a 3D seg specific visualization function.
+    """
+
+    def forward_test(self, points, img_metas, **kwargs):
+        """
```

Comment:
Need some brief introduction to this function.

Reply:
Add.